arc42/src/07_deployment_view.adoc

```diff
@@ -248,6 +248,13 @@ The CLI can echo the full schema instead of doing the actual migration, too. Thi
 $ java -jar sandbox-{acc-version}.jar create-schema --spring.datasource.url=$JDBC_URL --spring.datasource.username=$USR --spring.datasource.password=$PASS
 ----
 
+=== Reset Sandbox data
+In a sandbox environment it is a good pattern to reset the data periodically to the factory-defaults. To reset the data of the accelerator, the following steps are needed:
```

Comment:
please change "accelerator" to "PSD2 Accelerator" or "sandbox"

Reply:
Done